source/appModules/lync.py

```diff
@@ -18,15 +18,23 @@ def event_liveRegionChange(self):
 		# The object's name contains the full message (I.e. person: content, timestamp) loosely separated by commas.
 		# Example string: "Michael Curran : , , Hello\r\n\r\nThis is a test , 10:45 am."
 		# Where person is "Michael Curran", content is "Hello\nThis is a test" and timestamp is "10:45 am" 
-		# The object's value just contains the content.
-		# Example: "Hello\rThis is a test"
-		# We are only interested in person and content
-		# Therefore use value (content) to  locate and split off the person from the name (fullText)
-		# Normalize the usage of end-of-line characters (name and value seem to expose them differently, which would break comparison)
-		content=self.value.replace('\r','\n').strip()
-		fullText=self.name.replace('\r\n\r\n','\n')
+		fullText = self.name.replace('\r\n\r\n', '\n')
+		if self.value is None:
+			# For some versions of Skype for Business, self.value is just None.
+			# So we just look at self.name formatting to split content from person and timestamp.
+			contentStartIndex = self.name.find(', , ') + 4
+			contentEndIndex = self.name.rfind(', , ')
+			content = self.name[contentStartIndex:contentEndIndex].replace('\r\n\r\n', '\n').strip()

```

Comment:
could this be replaced with a regex match, and an logged error if this can't be found properly?